examples/index.html

```diff
@@ -575,18 +575,26 @@ <h3>Translating texts</h3>
         id: id,
         showAllValues: false,
         source: countries,
-        tStatusQueryTooShort: (minQueryLength) => `Tippe mindestens ${minQueryLength} Zeichen ein für Resultate.`,
-        tStatusNoResults: () => 'Keine Resultate.',
-        tStatusSelectedOption: (selectedOption, length) => `${selectedOption} (1 von ${length}) ist ausgewählt.`,
-        tStatusResults: (length, contentSelectedOption) => {
-          const words = {
+        tStatusQueryTooShort: function (minQueryLength) {
+          return 'Tippe mindestens ' + minQueryLength + ' Zeichen ein für Resultate.'
+        },
+        tStatusNoResults: function () {
+          return 'Keine Resultate.'
+        },
+        tStatusSelectedOption: function (selectedOption, length) {
+          return selectedOption + ' (1 von ' + length + ') ist ausgewählt.'
+        },
+        tStatusResults: function (length, contentSelectedOption) {
+          var words = {
             result: (length === 1) ? 'Resultat' : 'Resultate',
             is: (length === 1) ? 'ist' : 'sind'
           }
 
-          return `${length} ${words.result} ${words.is} verfügbar. ${contentSelectedOption}`
+          return length + ' ' + words.result + ' ' + words.is + ' verfügbar. ' + contentSelectedOption
         },
-        tNoResults: () => 'Keine Resultate'
+        tNoResults: function () {
+          return 'Keine Resultate'
+        }
```

Comment:
👍 

Non-blocking: It would be good to note somewhere that script in `/examples` files is not transpiled and should therefore use pre-ES6 syntax. Maybe a new entry "Examples" for  [CONTRIBUTING.md](https://github.com/alphagov/accessible-autocomplete/blob/master/CONTRIBUTING.md)?